### ISSUE TITLE
make doesn't install the fonts

### ISSUE THREAD

**Issue by rshopnil** [pcaro90/hermit#12]
I've downloaded the repo using `git clone` and used `make` to install the fonts. However, I'm getting this error

`cp: cannot stat 'Hermit*.otf': No such file or directory`
`make: *** [Makefile:13: install] Error 1`

After inspection, I see there're no .`otf `files present inside the project directory. How do I  solve this issue?

**Comment by pcaro90** [comment]
That's right, the `Makefile` is completely outdated. `otf` files are inside the `packages` folder.
